packages/nimble-components/src/utilities/unit-format/decimal-unit-format.ts

```diff
@@ -53,6 +54,37 @@ export class DecimalUnitFormat extends UnitFormat<DecimalUnitFormatOptions> {
         this.maximumFractionDigits = maximumFractionDigits;
     }
 
+    public static normalizeAndDefaultFractionDigitOptions(
+        fractionDigits?: number,
+        minimumFractionDigits?: number,
```

Comment:
I believe both callers pass `undefined` for `minimumFractionDigits`. Is that correct? Should it be removed from the function signature?

Reply:
Ultimately removed `minimumFractionDigits` parameter.